cEP-0024.md

```diff
@@ -0,0 +1,82 @@
+#Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+##Abstract
+
+This cEP describes the details of the process of creating new automations and
+plugins in GitMate which helps coala development workflow, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+GitMate.io is an automation tool for developers. While it works well, there are
+a number of plugins which saves a lot of time of our
+community(for example automatically close PRs from master). Many of them are
+general plugins, which are required by every org, but we need more plugins
+which support coala development workflow.
+
+## New Features
+
+### Auto-assign issue based on commit mention
+
+When a commit mentions(fixes,closes) an issue, auto assign unassigned issues.
+
+### Auto-reject PR opened on master of forks
+
+Normally, we should make changes in different branch from master of forks, and
+submit a PR. But, sometimes we receive PR’s from master of forks. To solve this
+problem, we need a mechanism by which gitmate automatically rejects those PR’s,
+and inform the PR author to work on a different branch.
+
+### Auto-mark PR with necessary labels as per the reviews done
+
+When a reviewer reviews a PR and either he approves changes or request changes
```

Comment:
'approves'/'requests changes' means `ack`/`nack` and/or GitHub review approvals/review changes requested & GitLab MR approvals / comments ?

Reply:
GitHub review approvals/review changes requested & GitLab MR approvals / comments

Reply:
@jayvdb ^